homeassistant/components/airzone/climate.py

```diff
@@ -39,6 +46,22 @@
 from .coordinator import AirzoneUpdateCoordinator
 from .entity import AirzoneZoneEntity
 
+BASE_FAN_SPEEDS: Final[dict[int, str]] = {
+    0: FAN_AUTO,
+    1: FAN_LOW,
+}
+FAN_SPEED_MAPS: Final[dict[int, dict[int, str]]] = {
+    2: BASE_FAN_SPEEDS
+    | {
+        2: FAN_HIGH,
+    },
+    3: BASE_FAN_SPEEDS
+    | {
+        2: FAN_MEDIUM,
+        3: FAN_HIGH,
+    },
```

Comment:
black really could do a better job formatting these type of unions, but nothing we can do about that.